indexd/index/schema.py

```diff
@@ -1,5 +1,5 @@
 POST_RECORD_SCHEMA = {
-    "$schema": "http://json-schema.org/schema#",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Can you send me the info you found on resolving this? Specifying a draft version is making my spidey-sense tingle.

Reply:
this resolved a couple hundred [warnings](https://gitlab.datacommons.io/nci-gdc/development/indexd/-/jobs/746153#L1302) while running pytest. Reading the warning message was sufficient to deduce the problem

Reply:
The short answer is that json-schema is not a finished specification and has no stable.